RHEL/6/xccdf/services/avahi.xml

```diff
@@ -26,9 +26,9 @@ port, it is subject to network attacks. Its functionality
 is convenient but is only appropriate if the local network
 can be trusted.
 </rationale>
-<ident cce="27087-6" stig="RHEL-06-000246" />
+<ident cce="27087-6" />
 <oval id="service_avahi-daemon_disabled" />
-<ref nist="CM-7" disa="366"/>
+<ref nist="CM-7" disa="366" stigid="RHEL-06-000246" srg="SRG-OS-999999" />
```

Comment:
Does RHEL6 use ``srg`` or ``ossrg``?

Reply:
@shawndwells it uses SRG. In fact, to make things simpler, I plan on getting rid of `ossrg` and `appsrg` and just make everything `srg` as that seems simpler.